Failure on aarch64-linux [(full log)](https://logs.nix.ci/?key=nixos/nixpkgs.36832&attempt_id=7d164c79-2bc2-4547-b095-e403b4d6bfb0)

Attempted: wxhexeditor

<details><summary>Partial log (click to expand)</summary><p>

```
             ^~~~~~~~
/nix/store/43rwn4kzfcg4i020cdmnijr7089dxk4x-wxwidgets-3.1.0/include/wx-3.1/wx/string.h:3267:13: note: candidate: wxString& wxString::operator+=(unsigned char)
   wxString& operator+=(unsigned char ch) { return *this += wxUniChar(ch); }
             ^~~~~~~~
/nix/store/43rwn4kzfcg4i020cdmnijr7089dxk4x-wxwidgets-3.1.0/include/wx-3.1/wx/string.h:3268:13: note: candidate: wxString& wxString::operator+=(wchar_t)
   wxString& operator+=(wchar_t ch) { return *this += wxUniChar(ch); }
             ^~~~~~~~
make: *** [Makefile:58: src/HexEditorCtrl/wxHexCtrl/wxHexCtrl.o] Error 1
builder for '/nix/store/wgylyy8fv1l248y3dziia504b4nsshqk-wxHexEditor-v0.24.drv' failed with exit code 2
[31;1merror:[0m build of '/nix/store/wgylyy8fv1l248y3dziia504b4nsshqk-wxHexEditor-v0.24.drv' failed
```
</p></details>

--- a/pkgs/applications/editors/wxhexeditor/default.nix
+++ b/pkgs/applications/editors/wxhexeditor/default.nix
@@ -1,4 +1,5 @@
-{ stdenv, fetchFromGitHub, wxGTK, autoconf, automake, libtool, python, gettext, bash }:
+{ stdenv, fetchFromGitHub, fetchpatch,
+wxGTK, autoconf, automake, libtool, python, gettext, bash }:
 
 stdenv.mkDerivation rec {
   name = "wxHexEditor-${version}";
@@ -19,6 +20,14 @@ stdenv.mkDerivation rec {
     substituteInPlace Makefile --replace "/usr" "$out"
     substituteInPlace Makefile --replace "mhash; ./configure" "mhash; ./configure --prefix=$out"
   '';
+
+  patches = [
+    # https://github.com/EUA/wxHexEditor/issues/90
+    (fetchpatch {
+      url = https://github.com/EUA/wxHexEditor/commit/d0fa3ddc3e9dc9b05f90b650991ef134f74eed01.patch;
+      sha256 = "1wcb70hrnhq72frj89prcqylpqs74xrfz3kdfdkq84p5qfz9svyj";
+    })
+  ];
 
   buildPhase = ''
     make OPTFLAGS="-fopenmp"

--- a/pkgs/applications/editors/wxhexeditor/default.nix
+++ b/pkgs/applications/editors/wxhexeditor/default.nix
@@ -1,5 +1,4 @@
-{ stdenv, fetchFromGitHub, fetchpatch,
-wxGTK, autoconf, automake, libtool, python, gettext, bash }:
+{ stdenv, fetchFromGitHub, fetchpatch, wxGTK, autoconf, automake, libtool, python, gettext, bash }:
 
 stdenv.mkDerivation rec {
   name = "wxHexEditor-${version}";

